app/controllers/ideas_controller.rb

```diff
@@ -56,6 +56,7 @@ def destroy
 
   def submit
     @idea.submission_date = Time.now
+    @idea.status = 0
```

Comment:
It might be more readable, and safe, to do `@idea.awaiting_approval` or `@idea.status = Idea.statuses[:awaiting_approval]` here.